x/delayedack/types/params.go

```diff
@@ -16,10 +16,13 @@ var (
 
 	// KeyBridgeFee is the key for the bridge fee
 	KeyBridgeFee = []byte("BridgeFee")
+
+	KeyDeletePacketBatchSize = []byte("DeletePacketBatchSize")
 )
 
 const (
-	defaultEpochIdentifier = "hour"
+	defaultEpochIdentifier       = "hour"
+	defaultDeletePacketBatchSize = 1000
```

Comment:
The default valuie should be much higher IMO
the goal of this value is to be **PROTECTION against memory exhasution**, not optimization tuning